README.md

```diff
@@ -110,3 +110,23 @@ gateway-url: https://api.my-env.openshift.com
 ### Full yaml config file
 
 See [example](./config.example.yaml)
+
+## Reporting
+
+To generate the report run the following command:
+
+`./export_report.py /tests/2021-05-18`
+
+The first argument should be the path to the folder where the `results` folder is located.
+
+### Graph a specific file
+
+`./export_report.py graph /tests/2021-05-18/results/access_review.json`
```

Comment:
(nitpick) I would make one minor change for how this script/module is executed in the documentatoin:

`python3 export_report.py ....`

The reason is that the script uses `/usr/bin/python3` when called directly, but many users (myself included) may actually run this within a Virtual Environment and I'm not entirely sure how this would impact the usage of the site-packages installed in that virtual environment.

Side Note: May also want to mention the installation of `requirements.txt`